resources/ctl.rb

```diff
@@ -101,13 +73,8 @@
   end
 end
 
-action :uninstall do
-  unless platform?('windows')
-    package 'sensu-go-cli' do
-      action :remove
-    end
-  end
-
+# This removes the folder we were previously putting the binary into.
+action :migrate do
```

Comment:
Should this have a guard/check to make it a safe to run? I imagine we only ever want that run once for an instance right? For example checking if `c:/sensutemp/` exists. Also thinking perhaps a more specific name for this would be good, we will eventually remove this in a later release anyways. How about something along the lines of?
```suggestion
action :cleanup_legacy_sensu_install do
```

Reply:
The reason we have this is because we were unpacking to a directory that this cookbook created, not Sensu. `sensuctl_bin` was intended to be the path to the executable, but somehow it got referenced as the directory, placing the executable one level lower than it should have been. There's nothing else that could have created that directory within the path, nor is there anything else that would create that directory aside from this cookbook. Even if this were to be run repeatedly, the resource deletion of both the path and the folder simply wouldn't do anything if they were already deleted.

Also, `c:/sensutemp` should not exist. We delete it here, after we've finished using it: https://github.com/sensu/sensu-go-chef/blob/master/resources/ctl.rb#L85-L88